shared/clientstats/types.go

```diff
@@ -0,0 +1,40 @@
+package clientstats
+
+type CommonStats struct {
+	CPUProcessSecondsTotal int64 `json:"cpu_process_seconds_total"`
+	MemoryProcessBytes int64 `json:"memory_process_bytes"`
+	ClientName string `json:"client_name"`
+	ClientVersion string `json:"client_version"`
+	ClientBuild int64 `json:"client_build"`
+	// NA
+	SyncEth2FallbackConfigured bool `json:"sync_eth2_fallback_configured"`
+	// NA
+	SyncEth2FallbackConnected bool `json:"sync_eth2_fallback_connected"`
+}
+
+type BeaconNodeStats struct {
+	// TBD -- pending merge of slasher refactor
```

Comment:
Let's add the format TODO(#ISSUENUM) as it is checked by a linter and is actionable to the reader connected to an actual issue thread. Might need gofmt as well

Reply:
https://github.com/prysmaticlabs/prysm/blob/develop/scripts/check-todo.sh linter in question